fluent-package/yum/confluent-test.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+set -exu
+
+if [ "$CI" = "true" ]; then
+   echo "::group::Setup confluent test"
+fi
+
+# Amazon Linux 2 system-release-cpe is:
+# cpe:2.3:o:amazon:amazon_linux:2
+# CentOS 7 system-release-cpe is:
+# cpe:/o:centos:centos:7
+# This means that column glitch exists.
+# So, we should remove before "o" character.
+
+distribution=$(cat /etc/system-release-cpe | awk '{print substr($0, index($1, "o"))}' | cut -d: -f2)
+version=$(cat /etc/system-release-cpe | awk '{print substr($0, index($1, "o"))}' | cut -d: -f4)
```

Comment:
Is it intended to use `$1` instead of `$0`?
Looks like the string does not include the delimiter of `awk` and `$1` would be the same as `$0`.

Reply:
Oh, it should be `$1`. Thanks.